### ISSUE TITLE
React: Masthead all the menu options which has sub options

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4198]
## Detailed description
On Voiceover iPhone
Issue: 
1. After expanding the focus is not set to the   first interactive element
, for screen reader users the focus stays on the triggering element  which is not visually present.
2. When Level 1 menu options are displayed, screen reader also identifies the content from the level 0 which is visually not present. this issue happens in all the menu levels as well.
3. After activation the sub options are collapsed visually but screen reader users are not able to navigate using left and right swipe.
, screen reader announces  only back.

**Comment by annawen1** [comment]
Hi @Praveen-sr, are these bugs still an issue? I think for the first item - wondering if that means the focus should immediately go to the first menu item?

**Comment by Praveen-sr** [comment]
@annawen1 Retested the masthead hamburger issue in react staging:
In Vo iPhone, Following fields are visually graded out when the hamburger menu is displayed. But Screen reader still identifies them,
IBM logo
Open IBM search field
User profile
The expected behavior is that the graded elements should be hidden from the screen reader as well.

**Comment by annawen1** [comment]
Because the new issue reported by Praveen is different form this ticket, I will close this ticket and open a new one with the new issue.
